server/main.go

```diff
@@ -136,6 +136,9 @@ func main() {
 	router.HandleFunc("/contributors/{login}", contributor.HandleGetContributor(database))
 	router.Post("/github/link", handler.HandleLink(database))
 
+	// Get last sync time
+	router.HandleFunc("/sync/last", handler.HandleGetLastSync(syncer))
```

Comment:
you can ignore this, just a feeling that having a route just for sync last is too much and that last sync should be in the route that returns the synced data.